lib/rules.js

```diff
@@ -196,6 +235,70 @@ export const PRODUCT_RULE = defineRuleString('#a^#b_0 * ...', '#a^(#b_0 + ...)')
 // e.g. x^5 / x^3 -> x^(5 - 3)
 export const QUOTIENT_RULE = defineRuleString('#a^#p / #a^#q', '#a^(#p - #q)')
 
+// e.g. x^2 * x^2 -> x^4
+export const MULTIPLY_POLYNOMIALS = defineRule(
```

Comment:
`mathsteps` has no `MULTIPLY_POLYNOMIALS`... it has a set of steps.  We should implement the steps in https://github.com/socraticorg/mathsteps/blob/master/lib/ChangeTypes.js#L60-L69 unless there's a strong reason why we shouldn't.